js/piwik.js

```diff
@@ -4689,7 +4619,7 @@ if (typeof window.Piwik !== 'object') {
                 } else if (windowAlias.addEventListener) {
                     windowAlias.addEventListener('load', callback);
                 } else if (windowAlias.attachEvent) {
-                    windowAlias.attachEvent('onLoad', callback);
+                    windowAlias.attachEvent('onload', callback);
```

Comment:
Should be all lowercase actually. Possibly `onLoad` works as well but lowercase is better